tests/testthat/test-a-recording.R

```diff
@@ -43,3 +43,62 @@ test_that("We can specify the path when starting capture", {
   dbDisconnect(con)
   stop_db_capturing()
 })
+
+test_that("dbGetQuery error checking", {
+  # Check to make sure con was not created before start_db_capturing()
+  suppressMessages(con <- nycflights13_create_sqlite(verbose = FALSE))
+
+  start_db_capturing()
+
+  # Testthat sets this to "tests/testthat//_memory_"
+  # Setting this to NULL so it will mimic a developers experience
+  .dittodb_env$db_path <- NULL
+
+  error_get_query <- expect_error(
+    object = dbGetQuery(con, "SELECT * FROM airlines"),
+    regexp = "^Database capture failed"
+  )
+  expect_equal(error_get_query$n, 1)
+
+  # The following functions trigger two errors
+  # testthat_transition( )
```

Comment:
```suggestion
  # testthat_transition( )
```

This is leftover yeah?

Reply:
fixed